dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -47,7 +47,7 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
 
             auto localBackgroundTask = winrt::create_instance<winrt::Windows::ApplicationModel::Background::IBackgroundTask>(m_comServerClsid, CLSCTX_ALL);
             auto pushBackgroundTaskInstance{ winrt::make<PushBackgroundTaskInstance>() };
```

Comment:
Check out C++/WinRT's `make_self` that returns you the `impl` type you can use, like this:

```c++
auto pushBackgroundTaskInstance { winrt::make_self<PushBackgroundTaskInstance>() };
pushBackgroundTaskInstance->SetRawNotificationPayload(payloadString);
localBackgroundTask.Run(*pushBackgroundTaskInstance);
```